cve_bin_tool/strings.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+
+"""
+These are the customized strings and file classes, by doing this
+the tool is able to support other operating systems like Windows
+and MacOS.
+"""
+
+import sys
+
+class Strings(object):
+
+    def __init__(self, filename):
+        self.filename = filename
+        self.o = ''
+
+    def parse(self):
+        if sys.version_info.major == 3:
+            return self.parse_3()
+        else:
+            return self.parse_2()
+
+    def parse_2(self):
+        f = file(self.filename, 'rb')
+        l = f.readline()
+        tmp = ''
+        while l:
+            for c in l:
+                if ord(c) < 128:
+                    tmp += c
+                else:
+                    if tmp != '':
+                        self.o += tmp + '\n'
+                        tmp = ''
+                    
+            l = f.readline()
+        return self.o
+
+    def parse_3(self):
```

Comment:
Wouldn't it be better if the parsing part of the code is made into a single function? For instance, once the file `f` is obtained, a single parse function can be called.

Reply:
I've thought about that, but the thing is python2 and python3 treat characters in f differently, in python2, it is raw character, while in python3, it is byte object.

Reply:
How about encoding the characters returned by `python2` to hex and then back to int?

`c = int(c.encode('hex'), 16)`

Reply:
Let's not worry about this until this PR has tests. We can refactor after it's merged